spec/index.bs

```diff
@@ -2264,9 +2264,14 @@ GPUSampler includes GPUObjectBase;
     : <dfn>\[[descriptor]]</dfn>, of type {{GPUSamplerDescriptor}}, readonly
     ::
         The {{GPUSamplerDescriptor}} with which the {{GPUSampler}} was created.
+
     : <dfn>\[[compareEnable]]</dfn> of type {{boolean}}.
     ::
         Whether the {{GPUSampler}} is used as a comparison sampler.
+
+    : <dfn>\[[filterEnable]]</dfn> of type {{boolean}}.
```

Comment:
nit: how about `isFiltering` to match GPUSamplerBindingType?

similarly isComparison above?

Reply:
Sounds great!